client-tez/src/test/java/org/apache/tez/common/RssTezUtilsTest.java

```diff
@@ -158,4 +163,25 @@ public void attemptTaskIdTest() {
     TezTaskAttemptID tezTaskAttemptID = TezTaskAttemptID.fromString(uniqueIdentifierToAttemptId);
     assertEquals(originalTezTaskAttemptID, tezTaskAttemptID);
   }
+
+  @Test
+  public void testParseRssWorker() {
+    Map<Integer, Set<ShuffleServerInfo>> rssWorker = new HashMap<>();
+    int shuffleId = 1001602;
+    String hostnameInfo = "localhost;1001602=172.19.193.152:19999+0_1_2_3,172.19.193.153:19999+2_3_4_5";
```

Comment:
What's meaning of `2_3_4_5`? Could we add some assertion for it?

Reply:
0_1_2_3 is consist of partition id.

and we add comment and

Reply:
Could we add some test cases to verify the correctness of partition id?

Reply:
done